docs/STYLE.md

```diff
@@ -165,20 +170,25 @@ We use comments for:
 - If something in a function is strange, or unintuitive, do leave a comment explaining what's going on. We use `//` for this.
 - We also use `//` for temporary comments above a function. Feel free to use `TODO:` in these if appropriate.
 - A bug should be commented with an `//! @bug Bug description` above the code that causes the bug.
+- For "fake matches", awful code that exists only to make the function's assembly match, mark these with `//! FAKE`
 
 ## What goes where
 
 This section mostly applies to actors.
 
 ### Functions
 
-All functions should go in the main C file in the same order as the assembly (the latter is required to match anyway). (We may make exceptions for particularly large files with a particular organisational structure, but we ask that you check on Discord first before doing this)
+All functions should go in the main C file in the same order as the assembly (the latter is required to match anyway). (We may make exceptions for particularly large files with a particular organizational structure, but we ask that you check on Discord first before doing this)
+
+### Prototypes
+
+Only put prototypes that are necessary. They should be placed in the main C file, unless they are used by something outside of the file, where they should be placed in the header.
```

Comment:
Do you mean the .h file instead of .c file here?

Reply:
I meant for actors, I guess I should make a separate note for code files to avoid confusion.